src/client/game/game.ts

```diff
@@ -269,11 +290,12 @@ export default class Game {
     // End server socket event handling
   }
 
-  private restart() {
+  private restart(token: string = "") {
```

Comment:
This has a default, because when a game is _over_, any player can restart it. The token gating only happens if someone tries to restart the game while the session is in progress.